src/windows/list.cpp

```diff
@@ -484,6 +482,7 @@ bool GetDetailData(DeviceDescriptor* device,
     }
 
     device->raw = "\\\\.\\PhysicalDrive" + std::to_string(deviceNumber);
+    device->device = device->raw;
```

Comment:
If `device->device` is now the same thing as `device->raw`, do the Windows examples in the README need updating again?